src/frontend/src/lib/components/hero/HeroSignIn.svelte

```diff
@@ -22,6 +22,5 @@
 </div>
 
 <ButtonIC on:click={async () => await signIn({})} testId="login-button">
```

Comment:
Instead of modifying it, maybe we can remove that `ButtonIc` and just use a standard primary button?

Reply:
right! good one! done